ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -718,14 +718,15 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips
-- as: aws-ipi-proxy-sts-f14-ui
+- as: aws-ipi-proxy-sts-regen-cert-f14-ui
```

Comment:
Shall we apply this to 4.16, too ?

Reply:
Sure, we can have

Reply:
BTW, just found this is a UI job.  Would you mind to use another job to add the cert tests ?